src/helpers/getResultStringForAdditionalPropertiesType.ts

```diff
@@ -0,0 +1,42 @@
+import { DataTypes, SwaggerProps } from '../types';
+import { getDescription } from './getDescription';
+import { parseRefType } from './parseRefType';
+
+export const getResultStringForAdditionalPropertiesType = ({
+    additionalProperties,
+    description,
+    propertyName,
+}: {
+    additionalProperties: any;
+    description?: string;
+    propertyName: string;
+}) => {
+    let res;
+
+    switch (additionalProperties.type) {
+        case DataTypes.Boolean:
+            res = DataTypes.Boolean;
+            break;
+        case DataTypes.Integer:
+            res = DataTypes.Number;
+            break;
+        case DataTypes.Number:
+            res = DataTypes.Number;
+            break;
+        case DataTypes.String:
+            res = DataTypes.String;
+            break;
+        case DataTypes.Array:
+            if (additionalProperties.items && additionalProperties.items[SwaggerProps.$ref]) {
+                res = `${parseRefType(additionalProperties.items[SwaggerProps.$ref].split('/'))}[]`;
+            } else {
+                res = `"// TODO: Something is wrong"`;
```

Comment:
I wouldnt recommend to put TODOs in code even if its for logging purpose

Reply:
I agree, but it was already the case and in this PR I focused on adding the support for additional properties type without dictionnary, without changing patterns.

I have upcoming PRs to improve on this.

Reply:
But at the end of the day, we will still want to write in the result file.
Something more like this ?
`// Error: DataTypes.Array without items or items ref is not supported`